react/README.md

```diff
@@ -29,6 +29,9 @@ This style guide is mostly based on the standards that are currently prevalent i
   - Always use JSX syntax.
   - Do not use `React.createElement` unless you’re initializing the app from a file that is not JSX.
   - [`react/forbid-prop-types`](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/forbid-prop-types.md) will allow `arrays` and `objects` only if it is explicitly noted what `array` and `object` contains, using `arrayOf`, `objectOf`, or `shape`.
+  - Consider not using React memo,useMemo and useCallback hooks in the code unless there is no way to write the code.
```

Comment:
why? These hooks are critically important and should often be used - see https://attardi.org/why-we-memo-all-the-things/